client/js/components/procedure/StatementSegmentsList/StatementSegmentsEdit.vue

```diff
@@ -11,8 +11,8 @@
   <div data-dp-validate="segmentsStatementForm">
     <dp-loading v-if="isLoading" />
 
-    <!-- if statement has segments, display segments -->
-    <template v-else-if="hasSegments">
+    <!-- if statement has segments and user has the permission, display segments -->
+    <template v-else-if="hasSegments && hasPermission('area_statement_segmentation')">
```

Comment:
why did You check that here and not in `hasSegments` ?

Reply:
Unfortunately this has been merged already but I will adjust it here: https://github.com/demos-europe/demosplan-core/pull/1255 which is also related to this PR but fixes also the loading of segments